.github/workflows/build_and_release.yml

```diff
@@ -200,6 +200,85 @@ jobs:
           asset_name: ${{ env.ARTIFACT_NAME }}
           asset_content_type: application/octet-stream
 
+  # Create the AppImage
+  AppImage:
+    name: Create the AppImage
+    needs:
+      - determine_version
+      - create_release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install the AppImage bundler and Performous deps
+        id: fetch_deps
+        run: |
+          wget -O appimage-builder-x86_64.AppImage https://github.com/AppImageCrafters/appimage-builder/releases/download/v1.0.0-beta.1/appimage-builder-1.0.0-677acbd-x86_64.AppImage
+          chmod +x appimage-builder-x86_64.AppImage
+          sudo mv appimage-builder-x86_64.AppImage /usr/local/bin/appimage-builder
+          sudo apt-get install -y --no-install-recommends git cmake build-essential gettext help2man libopenblas-dev libfftw3-dev libicu-dev libepoxy-dev libsdl2-dev libfreetype6-dev libpango1.0-dev librsvg2-dev libxml++2.6-dev libavcodec-dev libavformat-dev libswscale-dev libjpeg-dev portaudio19-dev libglm-dev libboost-filesystem-dev libboost-iostreams-dev libboost-locale-dev libboost-system-dev libboost-program-options-dev libssl-dev libcpprest-dev libportmidi-dev libopencv-dev libaubio-dev
+
+      - name: Checkout Git
+        id: checkout_git
+        uses: actions/checkout@v3
+        with:
+          path: performous
+
+      - name: Build the AppImage
+        id: build_appimage
+        run: |
+          cd performous
+          mkdir build
+          cd build
+          PACKAGE_VERSION=${{ needs.determine_version.outputs.complete_version }}
+          cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DENABLE_WEBSERVER=ON -DENABLE_WEBCAM=ON -DPERFORMOUS_VERSION=$PACKAGE_VERSION ..
+          make -j$(nproc) install DESTDIR=../AppDir
+          cd ..
+          appimage-builder --recipe AppImageBuilder.yml --skip-test
```

Comment:
Within `AppImageBuilder.yml` we're listing a version `1.x` this should become `$PACKAGE_VERSION` any chance we can override that value from the cli or by tokenizing it like on osx?

Reply:
Yeah, it looks like I'll have to do the tokenizing to fix it.

Reply:
It's yaml, and each operation is executed in the bash shell. So referencing the version should work :)